### PR TITLE
Make okhttp easier to find in IntelliJ's recents

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -1,4 +1,4 @@
-rootProject.name = 'okhttp-parent'
+rootProject.name = "🍓 okhttp"
```
Comment (swankjesse) [inline settings.gradle:1]: open to suggestions on alternatives!

Comment (JakeWharton) [reply settings.gradle:1]: You're creating an arms race now. How about just OkHttp? You can type and filter the projects so it's only one or two characters away as is.

Comment (swankjesse) [reply settings.gradle:1]: Not in the ‘open recent’ menu?

Comment (yschimke) [reply settings.gradle:1]: Probably okhttp, moving okhttp (directory) to okhttp-core and then reverting the artifactId in publishedArtifactId back to okhttp.

**Comment by yschimke** [comment]
Also maybe for yourself https://stackoverflow.com/questions/43558618/groups-of-projects-in-intellij-idea

**Review by yschimke — CHANGES_REQUESTED** [review]
It's like spitting in the face of https://github.com/square/Aardvark developers

**Pull request closed without merging**